notebooks/responsibleaidashboard/responsibleaidashboard-census-classification-model-debugging.ipynb

```diff
@@ -119,7 +119,7 @@
     "    # Append classifier to preprocessing pipeline.\n",
     "    # Now we have a full prediction pipeline.\n",
     "    pipeline = Pipeline(steps=[('preprocessor', feat_pipe),\n",
-    "                               ('model', LGBMClassifier())])\n",
+    "                               ('model', LGBMClassifier(random_state=0))])\n",
```

Comment:
jfmk: why is random_state added?

Reply:
It's impossible to assert anything if we don't fix the inherently random behavior of these ML techniques. Every execution will result in a different outcome. This fixes it since we provide the random state rather than it choosing a random state itself. People can easily remove or change it if they like, of course, but this makes the notebook reproducible.

Reply:
Just to be sure I'm adding @imatiach-msft in case this makes the experience bad for users.

Reply:
@romanlutz  yes this makes sense to me, sure!

Reply:
Thanks for confirming!